certified-connectors/Integration Toolbox/apiDefinition.swagger.json

```diff
@@ -0,0 +1,525 @@
+            "type": "string",
+            "x-ms-summary": "Delimiter",
+            "required": true,
+            "x-ms-visibility": "important"
+          },
+          {
+            "name": "hasHeaderRecord",
+            "in": "query",
+            "description": "The first line of the csv file contains a header with column names.",
+            "type": "boolean",
+            "enum": [
+              true,
+              false
+            ],
+            "x-ms-summary": "Has Header Record",
+            "required": true,
+            "x-ms-visibility": "important"
+          },
+          {
+            "name": "hasFieldsEnclosedInQuotes",
+            "in": "query",
+            "description": "The fields in the csv file are enclosed in quotes.",
+            "type": "boolean",
+            "enum": [
+              true,
+              false
+            ],
+            "x-ms-summary": "Has Fields Enclosed In Quotes",
+            "x-ms-visibility": "important",
+            "required": true
+          },
+          {
+            "name": "Body",
+            "in": "body",
+            "schema": {
+              "example": "firstname,lastname,email\r\nJohn,\"Doe,doe\",john@example.com\r\nJane,Doe,jane@example.com",
+              "x-ms-visibility": "important"
+            },
+            "description": "CSV data.",
+            "x-ms-visibility": "important",
+            "required": true
+          }
+        ],
+        "consumes": [
+          "text/plain"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful response."
+          }
+        }
+      }
+    },
+    "/fixedwidthtojson": {
+      "post": {
+        "operationId": "FixedWidthToJson",
+        "summary": "Fixed width to JSON",
+        "description": "Convert fixed width text to JSON.",
+        "parameters": [
+          {
+            "name": "fieldWidths",
+            "in": "query",
+            "description": "A comma separated list of integers containing the width of every field in the file.",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Field Widths",
+            "x-ms-visibility": "important"
+          },
+          {
+            "name": "hasHeaderRecord",
+            "in": "query",
+            "description": "The first line of the csv file contains a header with column names.",
+            "type": "boolean",
+            "enum": [
+              true,
+              false
+            ],
+            "x-ms-summary": "Has Header Record",
+            "required": true,
+            "x-ms-visibility": "important"
+          },
+          {
+            "name": "Body",
+            "in": "body",
+            "schema": {
+              "example": "firstname lastname  email               \r\nJohn      Doe       john@example.com    \r\nJane      Doe       jane@example.com    ",
```

Comment:
@lfalck Are spaces added in the example value is intentional? If not, please remove the spaces

Reply:
Yes, they are intentional since the operation takes fixed width text data. In the example firstname and lastname are 10 characters long while email is 20 characters.